modules/apps/frontend-js/frontend-js-web/test/liferay/util/cropregion/get_crop_region.es.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import dom from 'metal-dom';
+import getCropRegion from '../../../../src/main/resources/META-INF/resources/liferay/util/cropregion/get_crop_region.es';
+
+describe(
+	'Liferay.Util.getCropRegion',
+	() => {
+		it(
+			'should return an object if region receives every argument except height',
+			() => {
```

Comment:
1. This test seems wrong; we are setting width and it returns cropped value of NaN. Is this a bug in utility, misuse of utility, or incorrect test implementation?

1. We are missing a test for correct usage of utility: realistic sample of image and region values, coupled with realistic output.